events/d_1_identity_and_access_control copy.toml

```diff
@@ -85,9 +85,9 @@ description=""
 
 [[timeslots]]
 startTime="10:45"
-speakers=[]
-title="Open Slot"
-description=""
+speakers=["@expede","@matheus23"]
+title="User Controlled Authorization Network (UCAN)"
+description="UCAN is a capability-based access token combining DIDs, SPKI, and JWT designed fro P2P use cases"
```

Comment:
```suggestion
description="UCAN is a capability-based access token combining DIDs, SPKI, and JWT designed for P2P use cases."
```

Reply:
Sounds good!

Also yeah this is a 45-minute slot.. that may be a lot for "just" UCAN. Feel free to take this down to a 20- or 30-minute slot 👍